backend/src/database/config/config.ts

```diff
@@ -0,0 +1,29 @@
+import config from '../../config'
+
+// We have to manually parse database URL because sequelize-typescript requires explicit
+// connection parameters.
+const connectionConfig = {
+  host: config.get('dbHost'),
+  username: config.get('dbUsername'),
+  port: config.get('dbPort'),
+  password: config.get('dbPassword'),
+  database: config.get('dbName'),
+  dialect: config.get('dbDialect'),
+}
```

Comment:
hmm. I _think_ convict allows for [nested config](https://www.npmjs.com/package/convict#:~:text=db), so perhaps we could reorganise the config so that we can just do...
```suggestion
const connectionConfig = config.get('db')
```